### ISSUE TITLE
Response body missing in the exception

### ISSUE THREAD

**Issue by tgodfrey** [workos/workos-ruby#287]
When 400 series errors are encountered, the response body is suppressed from the exception which is thrown. A few fields are pulled out, but it would be helpful to include the full JSON response in the exception.

https://github.com/workos/workos-ruby/blob/9e47c7c70642415f4b10d812354f8739b2ded19e/lib/workos/client.rb#L90-L134

For example:
```
 when 400
        raise InvalidRequestError.new(
          message: json['message'],
          http_status: http_status,
          request_id: response['x-request-id'],
          code: json['code'],
          errors: json['errors'],
          body: json
        )
```

**Comment by tgodfrey** [comment]
This issue could be seen as a more general case of this issue: https://github.com/workos/workos-ruby/issues/263

**Comment by PaulAsjes** [comment]
Thanks for reporting! This should now be fixed in v5.0.0
